README.md

```diff
@@ -0,0 +1,41 @@
+# COVID-19 NZ Data Processing
+
+Script to automatically fetch, process, and plot the NZ covid-19 data from the Ministry of Health's website (https://health.govt.nz).
+
+Getting started:
+- 
+Dependencies:
+- pandas
+- requests
+- xlrd
+- beautifulsoup4
+- matplotlib
+- redis
```

Comment:
I would expect people using a virtual environment (using `virtualenv`, for example), so those depencies could/should be listed in a `requirements.txt` file, so that all we need to do is `pip install -r requirements.txt`? (see https://github.com/cellml/cellml-specification for example.)

Either way, it would be nice to have a command that one could just copy/paste.

Reply:
Ok, I can see that those requirements are listed in `setup.py`. So, not sure there is a point in having those here again...?